bin/webpack-dev-server.js

```diff
@@ -140,17 +140,14 @@ new Server(webpack(wpOpt), options).listen(options.port, options.host, function(
 	if(options.historyApiFallback)
 		console.log("404s will fallback to /index.html");
 
-	if (options.proxy) {
-		var paths = Object.keys(options.proxy);
-		paths.forEach(function (path) {
-			var target;
-			if (typeof options.proxy[path] === 'string') {
-				target = options.proxy[path];
-			} else {
-				target = options.proxy[path].target;
-			}
-			console.log('proxying '+path+' to '+target);
+	if (options.proxy && !Array.isArray(options.proxy)) {
+		options.proxy = Object.keys(options.proxy).map(function (path) {
+			return {
+				path: path,
+				target: typeof options.proxy[path] === "string" ? options.proxy[path] : options.proxy[path].target
+			};
```

Comment:
Yeah it's just for `console.log`. It's straight from #148, I'll just remove it then.